docs/en/ufs/S3.md

```diff
@@ -279,44 +249,92 @@ alluxio.underfs.s3.upload.threads.max=80
-user permission to the bucket and the object for access control. If the given S3 user does not have
-the access permissions to the specified bucket, a permission denied error will be thrown. When
-Alluxio security is enabled, Alluxio loads the bucket ACL to Alluxio permission on the first time
-when the metadata is loaded to Alluxio namespace.
+Read more [trouble-shooting guidance](https://aws.amazon.com/premiumsupport/knowledge-center/s3-troubleshoot-403/)
+for 403 error from AWS.
 
-### Mapping from S3 user to Alluxio file owner
+### File Owner and Group
 
-By default, Alluxio tries to extract the S3 user display name from the S3 credentials. Optionally,
+Alluxio file system infers the file owner from the metadata of S3 buckets and objects.
+Since there is no group in S3 ACL, the owner is reused as the group.
+
+By default, Alluxio extracts the display name of the AWS user.
+In case this display name is not available,
+Alluxio sets file owner to this AWS user's [canonical user ID](https://docs.aws.amazon.com/general/latest/gr/acct-identifiers.html).
+
+This canonical user id is typically a long string like `79a59df900b949e55d96a1e698fbacedfd6e09d98eacf8f8d5218e7cd47ef2be`,
+often inconvenient to read and use in practice.
+Optionally, property
 `alluxio.underfs.s3.owner.id.to.username.mapping` can be used to specify a preset S3 canonical id to
-Alluxio username static mapping, in the format "id1=user1;id2=user2".  The AWS S3 canonical ID can
-be found at the console `https://console.aws.amazon.com/iam/home?#/security_credentials`.
-Please expand the "Account Identifiers" tab and refer to "Canonical User ID".
+Alluxio username static mapping, in the format "id1=user1;id2=user2".
+For example, edit `alluxio-site.properties` to include
+
+```properties
+alluxio.underfs.s3.owner.id.to.username.mapping=\
+79a59df900b949e55d96a1e698fbacedfd6e09d98eacf8f8d5218e7cd47ef2be=john
+```
+
+This configuration helps Alluxio recognize all objects owned by this AWS account as owned by user `john` in Alluxio namespace.
+To find out the AWS S3 canonical ID,
+check the console `https://console.aws.amazon.com/iam/home?#/security_credentials`,
+expand the "Account Identifiers" tab and refer to "Canonical User ID".
 
-### Mapping from S3 ACL to Alluxio permission
+### File Permission
 
 Alluxio checks the S3 bucket read/write ACLs to determine the owner's permissions to a Alluxio
-file. For example, if the S3 user has read-only access to the underlying bucket, the mounted
-directory and files will be set with 0500 permissions. If the S3 user has full access to the underlying bucket,
-the mounted directory and files will be set with 0700 permissions.
+file.
+For example, if the AWS user has read-only access to the mounted bucket, the mounted
+directory and files will be set with `0500` permissions.
+If the AWS user has full access to the underlying bucket,
+the mounted directory and files will be set with `0700` permissions.
+
```

Comment:
my own question: does alluxio ignore if there are object specific permissions in the bucket? because i can set a up bucket with no permissions and selectively give access to certain object prefixes within the bucket

Reply:
based on the source code, it looks like so. double checking it with calvin who wrote it

Reply:
maybe worth mentioning if it is the case. something like `Note that Alluxio only checks bucket ACLs when determining filesystem permissions and does not check the ACLS of each individual object.`

Reply:
added